flight/PiOS/Common/pios_servo.c

```diff
@@ -161,6 +161,10 @@ void PIOS_Servo_SetMode(const uint16_t *out_rate, const int banks, const uint16_
 	TIM_TimeBaseStructure.TIM_ClockDivision = TIM_CKD_DIV1;
 	TIM_TimeBaseStructure.TIM_CounterMode = TIM_CounterMode_Up;
 	for (int i = 0; i < banks_found; i++) {
+		// Skip the bank if no outputs are configured
+		if (timer_banks[i].max_pulse == 0)
+			continue;
+
```

Comment:
Because of this, should we also revert the previous sparky change?  (#1072)

Thinking about it, probably not...

What I want to do for quixote is to adapt the CC3D/Sparky things-- instead of a boolean, to specify a **timer** that should be pegged to 1MHz.  And we can make that conditional on the use of PPM/PWM.